### PR TITLE
Convert to using the @covid-modeling/api package

### PR DESCRIPTION
**Pull request by aeisenberg**
Remove the copied code and install from npm.

### CODE REVIEW

File: components/simulation-form/InterventionPeriodSection.tsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
+import {input} from '@covid-modeling/api'
+import {Intensity} from '@covid-modeling/api/dist/src/model-input'
```
Comment (adityasharad) [inline components/simulation-form/InterventionPeriodSection.tsx:2]: Should this still use `dist`? Don't we expose `Intensity` in `input`?

File: package.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
   "license": "MIT",
   "dependencies": {
     "@azure/storage-blob": "^12.1.1",
+    "@covid-modeling/api": "0.8.0",
```
Comment (adityasharad) [inline package.json:18]: 0.8.1?

File: test/new-simulation-state.test.ts [hunk 3]
```diff
@@ -22,7 +22,7 @@ describe('new-simulation-state', () => {
       })
     })
 
-    it('should initialize state with two interventions of increasing intensity', () => {
+    it('should initialize state with two interventions of increasing input.Intensity', () => {
```
Comment (adityasharad) [inline test/new-simulation-state.test.ts:25]: Heh find-replace.
```suggestion
    it('should initialize state with two interventions of increasing intensity', () => {
```

File: test/new-simulation-state.test.ts [hunk 4]
```diff
@@ -116,16 +116,16 @@ describe('new-simulation-state', () => {
...
+            caseIsolation: input.Intensity.Aggressive,
+            socialDistancing: input.Intensity.Moderate
           }
         ]
       })
     })
 
     it('should initialize state with two interventions on the different dates, first overriding the second', () => {
-      // GathRestrict10 and GathRestrict10 are the same intensity, so our periods have not changed. Just use
+      // GathRestrict10 and GathRestrict10 are the same input.Intensity, so our periods have not changed. Just use
```
Comment (adityasharad) [inline test/new-simulation-state.test.ts:128]:
```suggestion
      // GathRestrict10 and GathRestrict10 are the same intensity, so our periods have not changed. Just use
```

File: components/simulation-form/InterventionPeriodSection.tsx (hunk 1 above)
Comment (aeisenberg) [reply components/simulation-form/InterventionPeriodSection.tsx:2]: Thanks that should change.

File: package.json (hunk 2 above)
Comment (aeisenberg) [reply package.json:18]: 0.8.0 and 0.8.1 are the same (just published differently).

Comment (aeisenberg) [reply package.json:18]: Though, I'll change it just to be sure.

**Pull request merged**